LDM-294.tex

```diff
@@ -16,14 +16,14 @@
 \addtohist{3.6}{2019-01-29}{Review product tree. Add release management subsection. Update DMCCB and DMSE team. Approved in \jira{RFC-561}}{G.~Comoretto}
 \addtohist{3.7}{2019-02-06}{Add text for LSP and Middleware mangers. Time allocation policy for institutional scientists, add admin names to org chart. Approved in \jira{RFC-572}}{W.~O'Mullane, L.~Guy}
 \addtohist{3.8}{2019-07-29}{Updates to Org charts, glossary, third party software, risk management, milestones. Approved in \jira{RFC-621}}{W.~O'Mullane, L.~Guy}
-\addtohist{}{2019-12-21}{SST org charts, middleware lead, builds and fixes  milestones. \jira{DM-22393}}{W.~O'Mullane}
+\addtohist{3.9}{2020-02-14}{SST org charts, middleware lead, builds and fixes  milestonesm update SST chart.\jira{RFC-666}, \jira{DM-22393}}{W.~O'Mullane}
```

Comment:
Typo: "milestonesm update" and SST org chart is now mentioned twice.

Reply:
fixed by john